content/post/data-warehouse-sla-p2.md

```diff
@@ -27,47 +28,61 @@ Here are four examples of services provided with increasing specificity:
 * Query speed: Query speed is a challenging aspect to quantify as your data warehouse most likely supports several different use cases in your data warehouse (types of data and queries). Average query run time is a good place to start, but you may need to create a more nuanced metric (e.g., X% of queries finish in <Y seconds)
-* Data reliability / quality: Certainly you aspire to produce correct numbers, but it may not be possible to ensure perfection. I recommend tiering KPIs with descending levels of reliability - from your "bullet proof" metrics down to the numbers you put in the data warehouse for someone, but have not QA'd yet. I also recommend investing in the [Data Quality Flywheel]({{< relref "data-dies-in-darkness.md" >}})
-* Ease of use / documentation: This can be a general commitment to deliver the data in a way that is simple and easy to use. However, you can also commit to producing detailed documentation (e.g., tool tips in the BI, data dictionary)
+* Data reliability / quality: Certainly you aspire to produce correct numbers, but it may not be possible to ensure perfection. I recommend tiering KPIs with descending levels of reliability - from your "bullet proof" metrics down to the numbers you put in the data warehouse for someone, but have not QA'd yet. I also recommend investing in the [Data Quality Flywheel]({{< relref "data-dies-in-darkness.md" >}}) to make data quality everyone's responsibility
+* Ease of use / documentation: This can be a general commitment to deliver the data in a way that is simple and easy to use. However, you can also commit to producing detailed documentation (e.g., tool tips in your business intelligence tool, data dictionary)
 
 While you should strive to build a robust analytics stack, you may never achieve the same reliability as your company's production level systems. For example, it is reasonable to expect that the systems that run the website of an ecommerce company will be more robust than their data warehouse. It costs a lot more when the website is down than when the data in the data warehouse is a few hours staler than usual. You could certainly build a robust data warehouse with production system level reliability - it all comes down to whether it makes sense to do this given your limited resources.
 
-The challenge, however, changes when you have [intelligent applications]({{< relref "data-platform.md" >}}) in production that utilize your data warehouse. In this case you must define and deliver on a significantly more robust SLA.
+The challenge, however, changes when you have [intelligent applications]({{< relref "data-platform.md" >}}) in production that utilize your data warehouse (e.g., automated post purchase emails powered by your data warehouse). In this case you must define and deliver on a significantly more robust SLA. Supporting production systems doesn't just require more resources, it requires an entirely different mentality.
```

Comment:
If you really want to shorten the post, I would consider removing this paragraph. The post doesn't lose anything if this is gone (even though I agree with its thesis).

Reply:
Good call. I think I'll leave this one in, because I think it is an important takeaway